test/test_filt.py

```diff
@@ -209,7 +209,7 @@ def test_body(self):
 
     def test_method(self):
         q = self.req()
-        s = self.resp()
+        self.resp()
```

Comment:
`s` is not used - that's the reason why autopep8 decided to remove it.
Do you want to keep it? For what reason?